deployment/terraform/securitygroups.tf

```diff
@@ -0,0 +1,29 @@
+data "aws_vpc" "vpc_lookup" {
+  tags = {
+    Name = "${var.organization}-apps-${var.environment}-${data.aws_region.current.name}-vpc"
```

Comment:
I kept this as is from the config repo, but we can also pass in a VPC ID instead and lookup from that. No need to expose naming.